docs/src/content/docs/guides/create-interactive-grant.mdx

```diff
@@ -89,22 +96,20 @@ Provided the end-user approves the grant, the authorization server:
 - Sends the end-user to your client’s previously defined `finish uri`. The means by which the server sends the end-user to the URI is out of scope, but common options include redirecting the end-user from a web page and launching the system browser with the target URI.
 - Secures the redirect by adding a unique hash, allowing your client to validate the `finish` call, and an interaction reference as query parameters to the URI.
 
-### 4. Request a grant continuation
-
-Add the authorization server’s interaction reference, as the `interact_ref` value, to the body of your [continuation request](/apis/auth-server/operations/post-continue/#request-body).
+#### Request a grant continuation
 
-<CodeBlock title="Example">
+Add the authorization server’s interaction reference, as the `interact_ref` value, to the body of your continuation request.
 
-```ts
-{
-    interact_ref: ad82597c-bbfa-4eb0-b72e-328e005b8689,
-},
-```
-
-</CodeBlock>
+<ChunkedSnippet
+  source='https://raw.githubusercontent.com/interledger/open-payments-snippets/main/grant/grant-continuation.ts'
+  chunk='3'
+/>
 
 Issue the request to the `continue uri` supplied in the authorization server’s initial grant response. For example:
 
 `POST https://auth.rafiki.money/continue/49fb4a97-f880-449c-b0e9-d3ac1c947b10`
 
-A successful [continuation response](/apis/auth-server/operations/post-continue/#responses) from the authorization server provides your client with an access token and other information necessary to continue the transaction.
+A successful continuation response from the authorization server provides your client with an access token and other information necessary to continue the transaction.
```

Comment:
I think we should also add a response example here

Reply:
Will do